compose-files/docker-compose-delhi-0.7.1-arm64.yml

```diff
@@ -0,0 +1,202 @@
+
+version: '2.1'
+volumes:
+  db-data:
+  log-data:
+  consul-config:
+  consul-data:
+
+services:
+
+  volume:
+    image: edgexfoundry/docker-edgex-volume-arm64:0.8.0
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+      
+  edgex-core-consul:
+    image: consul:1.4.0
+    ports:
+      - "8400:8400"
+      - "8500:8500"
+      - "8600:8600"
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - volume  
+
+  edgex-core-config-seed:
+    image: edgexfoundry/docker-core-config-seed-go-arm64:0.7.1
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - volume
+      - edgex-core-consul 
+
+  edgex-mongo:
+    image: edgexfoundry/docker-edgex-mongo-arm64:0.8.0
+    ports:
+      - "27017:27017"
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - volume
+
+  edgex-support-logging:
+    image: edgexfoundry/docker-support-logging-go-arm64:0.7.1
+    ports:
+      - "48061:48061"
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - edgex-core-config-seed
+      - edgex-mongo
+      - volume
+
+  edgex-support-notifications:
+    image: edgexfoundry/docker-support-notifications-go-arm64:0.7.1
+    ports:
+      - "48060:48060" 
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - edgex-support-logging
+
+  edgex-core-metadata:
+    image: edgexfoundry/docker-core-metadata-go-arm64:0.7.1
+    ports:
+      - "48081:48081"
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - edgex-support-logging
+
+  edgex-core-data:
+    image: edgexfoundry/docker-core-data-go-arm64:0.7.1
+    ports:
+      - "48080:48080"
+      - "5563:5563"
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - edgex-support-logging
+
+  edgex-core-command:
+    image: edgexfoundry/docker-core-command-go-arm64:0.7.1
+    ports:
+      - "48082:48082"
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - edgex-support-notifications
+
+# Perhaps the error is network related, it could work when the network is fixed
```

Comment:
what is this comment in reference to?

Reply:
It was a comment done for the rules-engine that was later removed from the compose. 
So so sorry for the comment mistakes

Reply:
No problem, just wanted to understand what this was in reference to.